libs/input-mapping/tests/Functional/DownloadTablesDefaultTest.php

```diff
@@ -490,4 +492,104 @@ public function testReadTablesDevBucket(): void
             new ReaderOptions(true)
         );
     }
+
+    #[NeedsTestTables(1)]
+    public function testReadTablesDevBucketProtectedBranch(): void
```

Comment:
Toto testuje to vypnutí devInputsDisabled v 
https://github.com/keboola/platform-libraries/pull/129/files#diff-7316865549dfb3f47ea37c93d72187a49ae60e54fedfd161c0e569de2e03d3baR89